forge/test/operators/utils/compat.py

```diff
@@ -0,0 +1,80 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Compatibility methods and datatypes for Forge
+
+import forge
+import torch
+
+from loguru import logger
+from typing import Optional, List
+
+from forge import ForgeModule, Module, VerifyConfig
+from forge.op_repo import TensorShape
+from forge.op.eval.common import compare_with_golden
+
+
+# TODO - Remove this class once TestDevice is available in Forge
+class TestDevice:
+
+    pass
+
+
+def get_dtype_for_df(dev_data_format: forge.DataFormat = None) -> torch.dtype:
+
+    dtype: torch.dtype
+
+    if dev_data_format is None:
+        dtype = None
+    else:
+        dtype: forge.dtype = torch.float32
+
+        # if dev_data_format in (
```

Comment:
Is this one redundant?

Reply:
The initial dtype logic removed from the PR since it was not fully working. It will be introduced later.